packages/sanity/src/core/versions/components/GlobalBundleMenu.tsx

```diff
@@ -0,0 +1,142 @@
+import {AddIcon, CheckmarkIcon} from '@sanity/icons'
+import {Box, Button, Flex, Menu, MenuButton, MenuDivider, MenuItem, Text} from '@sanity/ui'
+import {useCallback, useContext, useState} from 'react'
+import {useRouter} from 'sanity/router'
+
+import {VersionContext} from '../../../_singletons/core/form/VersionContext'
+import {type Bundle, type Version} from '../types'
+import {BUNDLES, LATEST} from '../util/const'
+import {CreateBundleDialog} from './dialog/CreateBundleDialog'
+import {VersionIcon} from './VersionIcon'
+
+export function GlobalBundleMenu(): JSX.Element {
+  const router = useRouter()
+
+  // eslint-disable-next-line no-warning-comments
+  // FIXME REPLACE WHEN WE HAVE REAL DATA
+  const bundles = BUNDLES
+
+  // eslint-disable-next-line no-warning-comments
+  // FIXME REPLACE WHEN WE HAVE REAL DATA
+  const {currentVersion, setCurrentVersion, isDraft} = useContext(VersionContext)
+  const [createBundleDialogOpen, setCreateBundleDialogOpen] = useState(false)
+
+  const handleBundleChange = useCallback(
+    (bundle: Version) => () => {
+      const {name} = bundle
+
+      if (name === 'drafts') {
+        router.navigateStickyParam('perspective', '')
+      } else {
+        router.navigateStickyParam('perspective', `bundle.${name}`)
+      }
+
+      setCurrentVersion(bundle)
```

Comment:
I know you are waiting here to have the store for this, but will we need a state to handle this ?
I think it could be derived from the router